docs/about.rst

```diff
@@ -99,6 +99,13 @@ Please adhere to the following guidelines:
   reason. As another example, do not add a test that makes dozens of concurrent
   requests to a public service such as docker hub.
 
+These next guidelines **are not** mandatory, but will match the formatting adopted
+in the existing code base.
+
+* Docstrings should not end with a blank line.
+* Docstrings should be line-wrapped at 79 characters.
```

Comment:
More generally, all code should be line-wrapped at 79 characters.

Reply:
What about the following statement?

Line length should be line-wrapped at 79 characters.

Reply:
Sounds great.